CMakeLists.txt

```diff
@@ -341,9 +341,9 @@ macro(add_flang_library name)
     add_dependencies( ${name} ${LLVM_COMMON_DEPENDS} )
   endif( LLVM_COMMON_DEPENDS )
 
-  llvm_config( ${name} ${LLVM_LINK_COMPONENTS} )
+  llvm_config( ${name} ${LLVM_LINK_COMPONENTS} 
   if(WIN32)
-    list(APPEND LLVM_COMMON_LIBS Dbghelp.lib)
+    list(APPEND LLVM_COMMON_LIBS Dbghelp.lib Ws2_32.lib)
```

Comment:
I'm wondering this is the correct place for this. How about `runtime/flang/CMakeLists.txt`?

Reply:
It's not the correct place, but I'm cmake-impaired.

Reply:
Do you have a line number or would you like to fix it?

Reply:
https://github.com/isuruf/flang/blob/windows/runtime/flang/CMakeLists.txt#L594 something like
```
target_link_libraries(flang_shared PRIVATE Ws2_32.lib)
target_link_libraries(flang_static PRIVATE Ws2_32.lib)
```